app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md

```diff
@@ -0,0 +1,306 @@
+      type: LoadBalancer
+      http:
+        enabled: true
+        servicePort: 8003
+      tls:
+        enabled: true
+        servicePort: 8446
+    ```
+
+5. Fill in the rest of the parameters as they fit your implementation. Use the comments in the sample file to guide you, and see the documentation on [Kong Enterprise parameters](https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-parameters) for more details.
+
+## Step 8. Deploy Kong Enterprise on Kubernetes
+The steps in this section show you how to install Kong Enterprise on Kubernetes using Helm.
+>Note: the following instructions assume that you're running Helm 3.
+
+{% navtabs %}
+{% navtab kubectl %}
+1. Run:
+    ```
+    $ helm install  my-kong kong/kong --namespace kong --values ~/values.yaml
+    ```
+    This may take some time.
+
+2. Check pod status:
+    ```
+    $ kubectl get pods -n kong
+    ```
+    Once migrations are complete and the `my-kong-kong-<ID>` pod is running, move on to the next section.
+{% endnavtab %}
+{% navtab OpenShift oc %}
+1. Run:
+    ```
+    $ helm install  my-kong kong/kong --namespace kong --values ~/values.yaml
+    ```
+    This may take some time.
+
+2. Check pod status:
+    ```
+    $ oc get pods -n kong
+    ```
+    Once migrations are complete and the `my-kong-kong-<ID>` pod is running, move on to the next section.
+{% endnavtab %}
+{% endnavtabs %}
+
+## Step 9. Finalize Configuration and Verify Installation
+{% navtabs %}
+{% navtab kubectl %}
+1. Run:
+    ```
+    $ kubectl get svc -n kong my-kong-kong-admin --output=jsonpath='{.status.loadBalancer.ingress[0].ip}'
+    ```
+
+2. Copy the IP address from the output, then edit the `values.yaml` file to add the following line under `env` section:
+
+    ```
+    admin_api_uri: <your-ip>
+    ```
+
+3. Clean up:
+    ```
+    $ kubectl delete jobs -n kong --all
+    ```
+
+4. Update Kong to use the changed `values.yaml`:
+    ```
+    $ helm upgrade my-kong kong/kong --namespace kong --values ~/values.yaml
+    ```
+
+6. Once the upgrade is complete, run:
+    ```
+    kubectl get svc -n kong
+    ```
+
+    In the output, the IP in the `EXTERNAL_IP` column is your access point for Kong features, including Kong Manager and Kong Dev Portal:
+
+    ```
+    NAME                          TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                            AGE
+    my-kong-kong-manager          LoadBalancer   10.96.61.116     10.96.61.116    8002:31308/TCP,8445:32420/TCP      24m
+    my-kong-kong-portal           LoadBalancer   10.101.251.123   10.101.251.123  8003:31609/TCP,8446:32002/TCP      24m
+    ```
+{% endnavtab %}
+{% navtab OpenShift oc %}
+1. Run:
+    ```
+    $ oc get svc -n kong my-kong-kong-admin --output=jsonpath='{.status.loadBalancer.ingress[0].ip}'
+    ```
+
+2. Copy the IP address from the output, then edit the `values.yaml` file to add the following line under `env` section:
+
+    ```
+    admin_api_uri: <your-ip>
```

Comment:
Ditto comment from line 239: we should add a note to encourage users to switch this to a hostname, and explain that they must do so before enabling RBAC.